src/app/selectors.ts

```diff
@@ -36,31 +27,6 @@ export const sessionIdSelector = (state: RootState) => {
   return state.app.sessionId
 }
 
-export const verificationPossibleSelector = (state: RootState): boolean => {
-  const centralPhoneVerificationEnabled = centralPhoneVerificationEnabledSelector(state)
-  if (centralPhoneVerificationEnabled) {
-    return true
-  }
-
-  const e164Number = e164NumberSelector(state)
-  const saltCache = e164NumberToSaltSelector(state)
-  const shouldUseKomenci = shouldUseKomenciSelector(state)
-  const { komenci } = verificationStatusSelector(state)
-  const hideVerification = hideVerificationSelector(state)
-  if (hideVerification) {
-    return false
-  }
```

Comment:
Shouldn't we keep this?

Reply:
there is an early return above where the rest of this logic isn't executed if CPV is enabled, so i think it is safe to remove?

Reply:
I guess the question is, do we want to still be able to use `hideVerification`? while leaving CPV enabled.

Reply:
hmm, i lean towards no. what do you think? 

the name is a little misleading, we're using this to disable the "start" button on the decentralised verification screen rather than hiding the verification step during onboarding. i struggle to think of a valid scenario where we'd want to remotely disable phone verification on the front end - we can stop accepting new mappings from the backend, so i'm thinking the only reason why we'd want separate control of the front end is if our backend service is severly broken / we want to hide errors on the front end... do you think we need to prepare for this kind of fallback?

Reply:
True, the name is bit misleading, at least for the onboarding part.
I think it's fine to remove it. We should also remove the flag then. But can be done in a separate PR of course :D

Reply:
cool, i've removed it!